File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/PotatoMash.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Food;

/* JADX INFO: loaded from: classes3.dex */
public class PotatoMash extends Food {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable, it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.food_potato_mash_name;
        this.idDescription = R.string.food_potato_mash_description;
        this.idImage = R.drawable.potato_mash;
        this.price = 9L;
        this.feedPower = 27;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/PotatoMash.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Food

class PotatoMash : Food() {
    override fun configureProperties() {
        idName = R.string.food_potato_mash_name
        idDescription = R.string.food_potato_mash_description
        idImage = R.drawable.potato_mash
        price = 9L
        feedPower = 27
    }
}
